thrimbletrimmer/scripts/edit.js

```diff
@@ -15,6 +15,10 @@ const PAGE_STATE = {
 	CONFIRMING: 3,
 };
 
+// References to Jcrop "stages" for the advanced thumbnail editor crop tool
+var videoFrameStage;
+var templateStage;
```

Comment:
nitpick: Prefer `let` for new variable declarations, unless we need the specific weirdness of `var` for some reason.
```suggestion
let videoFrameStage;
let templateStage;
```